modules/openxr/openxr_api.h

```diff
@@ -100,7 +100,7 @@ class OpenXRAPI {
 	XrFormFactor form_factor = XR_FORM_FACTOR_HEAD_MOUNTED_DISPLAY;
 	XrViewConfigurationType view_configuration = XR_VIEW_CONFIGURATION_TYPE_PRIMARY_STEREO;
 	XrReferenceSpaceType reference_space = XR_REFERENCE_SPACE_TYPE_STAGE;
-	XrEnvironmentBlendMode environment_blend_mode = XR_ENVIRONMENT_BLEND_MODE_OPAQUE;
+	// XrEnvironmentBlendMode environment_blend_mode = XR_ENVIRONMENT_BLEND_MODE_OPAQUE;
```

Comment:
Should this be removed? Or is this pending implementation?

Reply:
Pending implementation, I think it's part of the pass through implementation which @kisg is porting. Possibly I should ignore it in this PR.

Reply:
Owh I remember now, because it's unused CI complained about it